cmd/background-newdisks-heal-ops.go

```diff
@@ -449,7 +449,6 @@ func monitorLocalDisksAndHeal(ctx context.Context, z *erasureServerPools) {
 					err := healFreshDisk(ctx, z, disk)
 					if err != nil {
 						globalBackgroundHealState.setDiskHealingStatus(disk, false)
-						printEndpointError(disk, err, false)
```

Comment:
why is it redundant? @dormanze

Reply:
We shouldn't log `context.Canceled` thats all 

```
errors.Is(err, context.Canceled)
```

Is the only error that must be ignored not others.

Reply:
You are right. The log of errors in other scenarios is still needed.